src/Parser/Rogue/Subtlety/Modules/Features/ShadowBladesUptime.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatPercentage } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Enemies from 'Parser/Core/Modules/Enemies';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+
+class ShadowBladesUptime extends Analyzer {
+  static dependencies = {
+    enemies: Enemies,
+    combatants: Combatants,
+  };
+
+  statistic() {
+    const shadowDanceUptime = this.combatants.selected.getBuffUptime(SPELLS.SHADOW_DANCE_BUFF.id) / this.owner.fightDuration;
```

Comment:
It looks like you mixed up class name and statistic? Class is called ShadowBladesUptime, this is getting Shadow Dance uptime.

Reply:
fixed